### DISCUSSION TITLE
Restart at the year of model fails

### DISCUSSION THREAD

**Discussion by cassiemavs** [NREL/ReEDS-2.0#158] [Q&A: Execution] [answered]
Is there anyway that the model can be restarted at the year where it fails? For example, I set the running time from 2020 to 2050, while the model fails to find the optimal at 2048. After I did the debug and fixed the issue, can I restart the model from 2048 instead of the very begaining? If yes, which settings should I change?

**Comment by wesleyjcole** [comment, marked as answer]
There are a few ways to restart a run after it failed.  For a single run, the method I typically use is to copy the call_{case_name}.bat file and then delete everything prior to the year where I want to restart the model.  Then I just run this copied batch file, and it will pick things up and continue.  Because everything is self-contained in the run folder, you will need to make sure any adjustments you made to the model are made to the files within the specific runs directory, otherwise your changes will not take effect.

The restart happens using the .g00 files, so you can only restart for a solve year if you have the .g00 file from the prior solve year.  In the call to `gams d_solveoneyear.gms` you will see a `r=g00files\{case_name}_{year}`, which is the argument used to specify the prior .g00 file to read in.
